af04f855-58c4-4db3-a191-45fe33381679/Sets/af362a3a-4f60-4050-801e-0a7bb8dd58bf/set.xml

```diff
@@ -2927,7 +2927,7 @@ Combat Action: Sacrifice this attachment to exhaust a target enemy non-Elite arm
 			<property name="Faction"        value="Tau"/>
 			<property name="Type"           value="Army"/>
 			<property name="Loyalty"        value=""/>
-			<property name="Traits"         value="Scout. Ally."/>
+			<property name="Traits"         value="Scout. Ally. Kroot."/>
```

Comment:
Conquest didn't have any official errata, but this seems ok as long as an explanation is added to the end of the description

`Errata from Brad Andres: Add Kroot trait`

Reply:
Description from the commit ?